packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -236,27 +220,23 @@ export class CXLMarketingNavElement extends LitElement {
   _onOverlayOpen(e) {
     const overlay = e.target;
 
-    /**
-     * Vertically align context menu panels.
-     * Check if the child overlay is larger than the top level overlay.
-     *
-     * @since 2023.02.11
-     */
-    const overlays = document.querySelectorAll(
-      'vaadin-context-menu-overlay[theme="cxl-marketing-nav"]'
-    );
+    if (window.matchMedia(this._wideMediaQuery).matches) {
+      const target = this.shadowRoot.querySelector('#overlays-wrapper');
+      const firstOverlay = document.querySelector(
+        'vaadin-context-menu-overlay[theme="cxl-marketing-nav"]'
+      );
 
-    if (overlays.length > 1) {
-      const topLevelOverlay = overlays[0];
-      const listBox = overlay.querySelector('vaadin-context-menu-list-box');
-      const previousOverlay = overlays[overlays.length - 2];
-      const previousListBox = previousOverlay.querySelector('vaadin-context-menu-list-box');
+      if (target) {
+        target.style.top = target?.style?.top ? target.style.top : firstOverlay.style.top;
```

Comment:
I'm not sure how helpful the optional chaining is here. We already checked if `target` exists, and for the most part can assume `style` exists as well.

Reply:
It seemed initially necessary to always maintain the same position, but actually being without it was better indeed. Thank you.